scrapers/MindGeek.yml

```diff
@@ -36,6 +37,7 @@ sceneByURL:
       - transangels.com/scene/
       - transsensual.com/scene
       - trueamateurs.com/scene/
+      - noirmale.com/scene/
```

Comment:
```suggestion
```

Per the automated PR check validation error:

```
URLs for type `scene` should be unique, already exists on /sceneByURL/0/url/15
```

I'm not clear what the value of having two separate lists of URLs for the same scraper type (or if there even is one), but you don't want the same URL listed twice.

Reply:
the two separate lists are for `sceneScraper` and `scriptScraper`, where presumably the domains work with one or the other